### PR TITLE
minor fixes and wordsmithing

### PR DESCRIPTION
**Pull request by jhdalek55**
small changes on
adoptions.md,
alliance.md,
audits.md
design.md
governance.md
index.md
overview.md
participate.md
timeline.md

### CODE REVIEW

**Comment by jhdalek55** [comment]
The changes are all minor, except for a small bit of wordsmithing on the splash page and overview. I requested a review primarily for those two pieces. I figured you might want to take a look before I merged. 

Obviously there are still mechanical things to fix (like the logo and color). I'm generally content with the copy at this point, but I guess it comes back to the audience we are preparing this for. I was happy you took a more journalistic/less encyclopedic approach to the two introductory sections. It's much more engaging text and more appropriate for a web site. I think perhaps I was being overly cautious in framing the scope of Uptane implementations due to the restrictions on public acknowledments.

Let me know if you think any more large scale changes are called for. Otherwise, I guess I can take a run at fixing the formatting.

Lois

File: governance.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The following industry representatives serve in an advisory capacity for Uptane.
 * Andre Weimerskirch of Lear Corporation and University of Michigan Transportation Research Institute
 * Arthur Taylor of HERE Technologies
 * Cameron Mott of Southwest Research Institute
-* Gloria Danna of General Telecom Systems
+* Gloria Danna of Ford
```
Comment (JustinCappos) [inline governance.md]: I think we can't say Ford.

File: timeline.md [hunk 2]
```diff
@@ -9,28 +9,27 @@ css_id: timeline
...
 to begin work on a project to secure software updates on vehicles.
 
-**2016**: Uptane is created when a consortium of researchers at 
-the [University of Michigan Transportation Research Institute](http://www.umtri.umich.edu/),
-[Southwest Research Institute](https://www.swri.org/) (SWRI), and 
+**2016**: Uptane is created when a consortium of researchers at
+the [University of Michigan Transportation Research Institute](http://www.umtri.umich.edu/)(UMTRI),
+[Southwest Research Institute](https://www.swri.org/)(SWRI), and
 [NYU Tandon](https://engineering.nyu.edu/)
-begin developing Uptane, using the basic TUF design as a starting point.
```
Comment (JustinCappos) [inline timeline.md]: You have a lot of these types of edits throughout.  Can you please clean them up?  No need to add spaces at the end of lines.

File: governance.md (hunk 1 above)
Comment (jhdalek55) [reply governance.md]: Ok. I'll put it back as is.

File: timeline.md (hunk 2 above)
Comment (jhdalek55) [reply timeline.md]: Did the best I could to catch these added breaks.

**Pull request merged**